test/unittest/base/test_datalayer.py

```diff
@@ -461,38 +436,35 @@ def test_insert_artifacts(db):
             [Document({'x': numpy.random.randn(100)}) for _ in range(1)]
         )
     )
-    r = db.execute(db['documents'].find_one())
+    r = list(db.execute(db['documents'].select()))[0]
     assert isinstance(r['x'], numpy.ndarray)
 
 
-@pytest.mark.parametrize("db", [DBConfig.sqldb_empty], indirect=True)
 def test_insert_sql_db(db):
+    db.cfg.auto_schema = True
```

Comment:
Maybe we should think about making `auto_schema` true by default in tests also. (For later.)